GDSCommon-Demo/GDSCommon-Demo/Mocks/Screens.swift

```diff
@@ -58,6 +59,9 @@ enum Screens: String, CaseIterable {
             return DatePickerScreenViewController()
         case .gdsTextInput:
             return TextInputViewController()
+        case .gdsLinkTheApp:
+            let viewModel = MockIconOptionsViewModel()
+            return IconOptionsViewController(viewModel: viewModel)
```

Comment:
This is a small block of code, but might be worth considering making an extension with a convenience init for this view controller to keep this switch statement small.

Reply:
I'll have a look at adding this 🙂